spec/cordova/emulate.spec.js

```diff
@@ -99,11 +99,15 @@ describe('emulate command', function () {
                 platformApi.build = originalBuildSpy;
             });
             it('Test 006 : should leave parameters unchanged', function () {
-                return cordova.run({ platforms: ['blackberry10'], options: { password: '1q1q' } })
+                const baseOptions = { password: '1q1q', device: false, emulator: true };
+                const expectedRunOptions = Object.assign({ nobuild: true }, baseOptions);
+                const expectedBuildOptions = Object.assign({ couldBeModified: 'insideBuild' }, baseOptions);
+
+                return cordova.emulate({ platforms: ['blackberry10'], options: { password: '1q1q' } })
```

Comment:
Do we still want to keep the deprecated  `blackberry10` reference or should we change it to a non-deprecated platform?

Reply:
I don't think it matters at all for these tests. If we were to change it, I'd use a platform that does not exist at all. A lot of tests use `atari` as a platform for that purpose.

Reply:
That would work :D

Was just a bit confusing cause I knew it was a valid platform at one point.

Anyways 👍